### ISSUE TITLE
`ValidationError: Invalid relations` when importOnBootstrap enabled (since v1.2.0)

### ISSUE THREAD

**Issue by sidemt** [boazpoolman/strapi-plugin-config-sync#109]
### Describe the bug

I'm using this plugin for setting up the test environment. But since v1.2.0 it started failing to import config with `ValidationError: Invalid relations`.

I can only reproduce this error when I import config with `importOnBootstrap: true`. If I use `yarn cs import` or `npm run cs import` manually, it works.

### Steps to reproduce the behavior

1. Clone [this repository](https://github.com/sidemt/config-sync-test)
2. run `yarn install`
3. run `yarn test`
4. See error `ValidationError: Invalid relations`

Error log:
```
$ yarn test
yarn run v1.22.19
$ jest --forceExit --detectOpenHandles
jest-haste-map: duplicate manual mock found: index
  The following files share their name; please delete one of them:
    * <rootDir>/.cache/admin/src/hooks/useAdminRolePermissions/__mocks__/index.js
    * <rootDir>/.cache/admin/src/hooks/useAdminRoles/__mocks__/index.js

jest-haste-map: duplicate manual mock found: index
  The following files share their name; please delete one of them:
    * <rootDir>/.cache/admin/src/hooks/useAdminRoles/__mocks__/index.js
    * <rootDir>/.cache/admin/src/hooks/useAdminUsers/__mocks__/index.js

jest-haste-map: duplicate manual mock found: index
  The following files share their name; please delete one of them:
    * <rootDir>/.cache/admin/src/hooks/useAdminUsers/__mocks__/index.js
    * <rootDir>/.cache/admin/src/hooks/useContentTypes/__mocks__/index.js

 FAIL  tests/app.test.js (7.376 s)
  ✕ strapi is defined

  ● strapi is defined

    Error when trying to import core-store.core_admin_auth. ValidationError: Invalid relations

       6 | async function setupStrapi() {
       7 |   if (!instance) {
    >  8 |     await Strapi().load();
         |     ^
       9 |     instance = strapi;
      10 |
      11 |     await instance.server.mount();

      at Object.importSingleConfig (node_modules/strapi-plugin-config-sync/server/services/main.js:233:13)
      at node_modules/strapi-plugin-config-sync/server/services/main.js:183:7
          at async Promise.all (index 3)
      at Object.importAllConfig (node_modules/strapi-plugin-config-sync/server/services/main.js:175:5)
      at Object.<anonymous>.module.exports [as bootstrap] (node_modules/strapi-plugin-config-sync/server/bootstrap.js:54:7)
      at Object.bootstrap (node_modules/@strapi/strapi/src/core/domain/module/index.ts:81:7)
      at Object.bootstrap (node_modules/@strapi/strapi/src/core/registries/modules.ts:29:9)
      at Strapi.runLifecyclesFunctions (node_modules/@strapi/strapi/src/Strapi.ts:639:5)
      at Strapi.bootstrap (node_modules/@strapi/strapi/src/Strapi.ts:610:5)
      at Strapi.load (node_modules/@strapi/strapi/src/Strapi.ts:619:5)
      at setupStrapi (tests/helpers/strapi.js:8:5)
      at Object.<anonymous> (tests/app.test.js:5:3)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        7.436 s, estimated 8 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Expected behavior

Can import config and run tests successfully.

### Screenshots

![image](https://github.com/boazpoolman/strapi-plugin-config-sync/assets/25644062/2d86102d-75c2-4f1b-af45-1a1ba89e5fa2)

### Code snippets

[Minimal code that can reproduce this issue](https://github.com/sidemt/config-sync-test)
It's created with the following steps:
- Run `npx create-strapi-app@latest config-sync-test --quickstart`
- Install config-sync
- Set up test environment according to: https://docs.strapi.io/dev-docs/testing
- Set `importOnBootstrap: true` in `config/env/test/plugins.js`

### System

- Node.js version: v18.16.0
- NPM version: 9.5.1
- Strapi version: 4.14.4
- Plugin version: 1.2.1
- Database: better-sqlite3 v8.6.0
- Operating system: Ubuntu 20.04.6 LTS (WSL2)

**Comment by boazpoolman** [comment]
Hi @sidemt,
Thanks submitting this issue and providing an example repo. That proved to be very helpfull.

I've been able to replicate this issue and directly wrote an extra integration test for the `importOnBootsrap` feature.
By doing that I've been able to reproduce this issue in a [failed pipeline](https://github.com/boazpoolman/strapi-plugin-config-sync/actions/runs/6571166649/job/17849828693#step:7:28).
And I was able to confirm the fix by observing the [successful pipeline](https://github.com/boazpoolman/strapi-plugin-config-sync/actions/runs/6578301696/job/17871751834).
This will hopefully prevent this issue from going unnoticed in the future.

The fix is included in the PR #111
Could you test that on your side?
You can simply install the PR like so:

```
yarn add boazpoolman/strapi-plugin-config-sync#pull/111/head
```

**Comment by MSACC** [comment]
Fixed in https://github.com/boazpoolman/strapi-plugin-config-sync/pull/111

**Comment by boazpoolman** [comment]
Re-opened.
Can be closed as soon as it's been released.

**Comment by boazpoolman** [comment]
The fix has been released with [v1.2.2](https://github.com/boazpoolman/strapi-plugin-config-sync/releases/tag/1.2.2)

If you're still having this, or any other issue with the plugin beyond v1.2.2 please feel free to open a new issue.
